### PR TITLE
Part 2 design using html css

### PR DESCRIPTION
**Pull request by araraltawarah98**

### CODE REVIEW

File: task_1/index.html [hunk 1]
```diff
@@ -11,34 +11,151 @@
...
           <p>Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain
             Coffea species. When coffee berries turn from.</p>
           <p>Top Packs</p>
         </div>
         <div class="coffee-list">
-          <div class="coffee-item"></div>
-          <div class="coffee-item"></div>
-          <div class="coffee-item"></div>
-          <div class="coffee-item"></div>
+          <div class="coffee-item">
```
Comment (abood454) [inline task_1/index.html:56]: the cards should have border 
and the text should be bigger

File: task_1/index.js [hunk 2]
```diff
@@ -1 +1,11 @@
-console.log("true");
+const onMenuItemClick = (elementId) => {
```
Comment (abood454) [inline task_1/index.js:1]: I think there is a bug in "our lands".
if you turn off our lands button and you try to turn it on again, the button will stop working.

File: task_1/style.css [hunk 3]
```diff
@@ -5,55 +5,179 @@
```
Comment (abood454) [inline task_1/style.css:5]: the alphabetical order is missing

**Review by abood454 — CHANGES_REQUESTED** [review]
The Design is good but I think the spaces between elements and elements sizes can be more accurate.

File: task_1/index.html [hunk 4]
```diff
@@ -11,34 +11,151 @@
...
+        <img src="./assets/images/Double espresso-1.png" alt="">
+        <h4>Double Espresso</h4>
+        <p>
+          Coffee is a brewed drink prepared
+          from roasted coffee beans, the seeds of
+          berries from certain Coffea species.
+        </p>
+      </div>
+      <div class="coffee-item">
+        <img src="./assets/images/Double espresso-1.png" alt="">
```
Comment (ashrafabuodeh) [inline task_1/index.html:142]: write alternative for image if the image not visible

File: task_1/index.js [hunk 5]
```diff
@@ -1 +1,11 @@
...
+  const itemId = document.getElementById(elementId);
+  let items = document.getElementsByClassName("menu-item");
+  const active = "active";
+  for (var i = 0; i < items.length; i++) {
+    if (items[i].className.includes(active) === true) {
+      items[i].classList.remove(active);
+    }
+  }
+  itemId.classList.add(active);
+};
```
Comment (ashrafabuodeh) [inline task_1/index.js:11]: add space here

File: task_1/style.css [hunk 6]
```diff
@@ -5,55 +5,179 @@
 
 body {
   font-family: Google sans;
+  background: linear-gradient(180deg, rgba(228, 239, 233, 0) 0%, #ecf8f1 6.25%);
 }
```
Comment (ashrafabuodeh) [inline task_1/style.css:10]: remove space

**Review by ashrafabuodeh — CHANGES_REQUESTED** [review]

File: task_1/style.css [hunk 7]
```diff
@@ -5,55 +5,179 @@
...
-  border: 1px solid black;
+  padding-top: 55px;
   min-width: 320px;
+  max-width: 420px;
+}
+
+.hero-image > img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
```
Comment (Mohsen-Awad) [inline task_1/style.css:84]: use shorthand margin

**Review by Mohsen-Awad — CHANGES_REQUESTED** [review]

File: task_1/index.js [hunk 8]
```diff
@@ -1 +1,11 @@
-console.log("true");
+const onMenuItemClick = (elementId) => {
+  const itemId = document.getElementById(elementId);
+  let items = document.getElementsByClassName("menu-item");
+  const active = "active";
+  for (var i = 0; i < items.length; i++) {
+    if (items[i].className.includes(active) === true) {
```
Comment (mmasaid) [inline task_1/index.js:6]: There is no need to mention === true _here_ 
```suggestion
    if (items[i].className.includes(active)) {
```

File: task_1/index.html [hunk 9]
```diff
@@ -11,34 +11,151 @@
...
-      <div class="coffee-item"></div>
-      <div class="coffee-item"></div>
-      <div class="coffee-item"></div>
-      <div class="coffee-item"></div>
-      <div class="coffee-item"></div>
-      <div class="coffee-item"></div>
-      <div class="coffee-item"></div>
-      <div class="coffee-item"></div>
+      <div class="coffee-item">
+        <img src="./assets/images/MOcha Late 02.png" alt="">
```
Comment (laithyounes) [inline task_1/index.html:88]: change the image `src` and image name here,
this will freakout on some OS/browsers

Comment (laithyounes) [reply task_1/index.html:88]: and please add `alt`

**Review by laithyounes — CHANGES_REQUESTED** [review]

**Review by abood454 — APPROVED** [review]

**Review by ashrafabuodeh — APPROVED** [review]

**Review by Mohsen-Awad — APPROVED** [review]

File: task_1/index.js [hunk 10]
```diff
@@ -1 +1,11 @@
-console.log("true");
+const onMenuItemClick = (elementId) => {
+  const itemId = document.getElementById(elementId);
```
Comment (mmasaid) [inline task_1/index.js:2]:
```suggestion
  const navItem = document.getElementById(elementId);
```

**Review by mmasaid — APPROVED** [review]
LGTM

**Pull request merged**